bindings/python/manifold3d.cpp

```diff
@@ -254,6 +254,31 @@ PYBIND11_MODULE(manifold3d, m) {
           "the user to choose their function with discretion."
           "\n\n"
           ":param warpFunc: A function that modifies a given vertex position.")
+      .def(
+          "set_properties",
+          [](Manifold &self, int newNumProp,
+             const std::function<void(py::array_t<float> &, Float3,
+                                      const py::array_t<float> &)> &f) {
+            const int oldNumProp = self.NumProp();
+            return self.SetProperties(newNumProp, [newNumProp, oldNumProp, &f](
+                                                      float *newProps,
+                                                      glm::vec3 v,
+                                                      const float *oldProps) {
+              f(py::array(newNumProp, newProps), std::make_tuple(v.x, v.y, v.z),
+                py::array(oldNumProp, oldProps));
```

Comment:
I'm getting a compile error here that I don't understand. Am I missing something?